test/integration/targets/nxos_devicealias/tasks/nxapi.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
`nxapi.yaml` seems to be commented out in tasks/main.yaml. In that case, this file is not required.